docs/source/app-dev/bindings-scala/index.rst

```diff
@@ -0,0 +1,144 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Scala bindings
+##############
+
+This tutorial provides a basic Scala programmer's introduction to working with Digital Asset distributed ledger,
+using the Scala programming language and the *Ledger API*.
+
+Introduction
+============
+
+The Scala bindings is a client implementation of the *Ledger API*. The Scala bindings library lets you write applications that connect to the Digital Asset distributed ledger using the Scala programming language.
+
+There are two main components:
+
+- Scala codegen
+    DAML to Scala code generator. Use this to generate Scala classes from DAML models. The generated Scala code provides a type safe way of creating contracts (:ref:`com.digitalasset.ledger.api.v1.createcommand`) and exercising contract choices (:ref:`com.digitalasset.ledger.api.v1.exercisecommand`).
+
+- Akka Streams-based API
+    The API that you use to send commands to the ledger and receive transactions back.
+
+In order to use the Scala bindings, you should be familiar with:
+
+- :doc:`DAML language </daml/reference/index>`
+- :doc:`Ledger API </app-dev/grpc/index>`
+- `Akka Streams API <https://doc.akka.io/docs/akka/current/stream/index.html>`_
+- `Scala programming language <https://www.scala-lang.org>`_
+- :ref:`assistant-manual-building-dars`
+
+Getting started with sbt
+========================
+
+Set up the following dependencies in your ``build.sbt``:
+
+.. literalinclude:: ./code-snippets/quickstart-scala/build.sbt
+   :start-after: // <doc-ref:dependencies>
+   :end-before: // </doc-ref:dependencies>
+
+We recommend separating generated code and application code into different modules. There are two modules in the example below:
+
+- ``scala-codegen``
+    This modules contains all generated Scala classes.
+- ``application``
+    This is the application code that makes use of the generated Scala classes.
+
+.. literalinclude:: ./code-snippets/quickstart-scala/build.sbt
+   :start-after: // <doc-ref:modules>
+   :end-before: // </doc-ref:modules>
+
+``scala-codegen`` module uses the following function to generate Scala classes from a DAR file.
+
+.. literalinclude:: ./code-snippets/quickstart-scala/build.sbt
+   :start-after: // <doc-ref:generate-scala>
+   :end-before: // </doc-ref:generate-scala>
+
+Here is the entire `build.sbt file <https://github.com/digital-asset/daml/blob/master/language-support/scala/examples/quickstart-scala/build.sbt>`_
```

Comment:
```suggestion
You can get the entire `build.sbt file <https://github.com/digital-asset/daml/blob/master/language-support/scala/examples/quickstart-scala/build.sbt>`__ from the ``daml`` repository on GitHub.
```

I'm a bit anxious about this link - if you move or rewrite the example, it'll just break.

Reply:
we should not be moving it. I thought the RST doc build tool actually checks if links exist

Reply:
It checks if links internal to the docs exist. It doesn't have a way of checking links to external websites!

Reply:
I will see if I can change it to point to the symlink inside the docs that I created

Reply:
I don't think that'll work - as far as I'm aware, that code file won't be published, so you won't be able to link to it within the docs site.

You might be able to make it a download link instead? Search "sphinx download directive" for how to do that - you'll want to preview it locally and make sure it works, though

Reply:
In this case I will leave it for later as a tech debt (issue will be created)... I have a hard deadline to finish the docs by Thursday. We will not be moving the examples anywhere any time soon.

Reply:
#1134